### PR TITLE
MDE XSUP-44022 - Correlation Rule Update

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-44022)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-  action_local_ip: xdm.target.ipv4
-  action_remote_ip:
-  action_remote_port:
-  actor_effective_username: xdm.source.user.username
+  eventid: xdm.event.id
 alert_name: Malware detected by Microsoft Defender for Endpoints
 crontab: '*/10 * * * *'
 dataset: alerts
-description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
+description: Alert will trigger an event when Malware was detected by Microsoft Defender for Endpoints.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml:15]:
```suggestion
description: Alert will trigger an event when Malware ks detected by Microsoft Defender for Endpoints.
```

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_17_6.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Correlation Rules
+
+##### Microsoft Defender for Endpoint - Malware Detected
+
+Updated the Correlation Rule logic, added an Alert Field mapping for the **Event ID** field to the **xdm.event.id** field.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_17_6.md:6]:
```suggestion
Updated the Correlation Rule logic. Added an Alert Field mapping for the *Event ID* field to the *xdm.event.id* field.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

File: Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
-  action_local_ip: xdm.target.ipv4
-  action_remote_ip:
-  action_remote_port:
-  actor_effective_username: xdm.source.user.username
+  eventid: xdm.event.id
 alert_name: Malware detected by Microsoft Defender for Endpoints
 crontab: '*/10 * * * *'
 dataset: alerts
-description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
+description: Alert will trigger an event when Malware ks detected by Microsoft Defender for Endpoints.
```
Comment (OBavly) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml:15]: I think you have a typo here + I would align this with the alert_description field.
I would change this sentence grammatically, as well - you are missing some determiners and prepositions (an + in).
I would change to "An alert will trigger" or "This alert will trigger" and "in an event", etc.

**Review by OBavly — CHANGES_REQUESTED** [review]

File: Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml (hunk 3 above)
Comment (eepstain) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml]: @OBavly Thank you!
Updated the description.

File: Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml [hunk 4]
```diff
@@ -1,24 +1,22 @@
...
-  action_local_ip: xdm.target.ipv4
-  action_remote_ip:
-  action_remote_port:
-  actor_effective_username: xdm.source.user.username
+  eventid: xdm.event.id
 alert_name: Malware detected by Microsoft Defender for Endpoints
 crontab: '*/10 * * * *'
 dataset: alerts
-description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
+description: An alert will trigger in an event when a Malware was detected by Microsoft Defender for Endpoint.
```
Comment (OBavly) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml:15]: Sorry for the hassle, but I think the best thing to write is:
This alert will trigger when Malware is detected by Microsoft Defender for Endpoint.
And I would copy the same to alert_description.

**Review by OBavly — CHANGES_REQUESTED** [review]

File: Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml (hunk 4 above)
Comment (eepstain) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml]: @OBavly Done!
Ready for re-approval.

**Review by OBavly — APPROVED** [review]

**Pull request merged**